Gemfile

```diff
@@ -34,14 +34,21 @@ else
   end
 
   case ENV['RAILS_VERSION']
+  when '6.0.0' # This will only be necessary until release 6.0.1 is published
+    gem 'sass-rails', '~> 5'
+    gem 'webpacker' # This is necessary or the generator will break
```

Comment:
This shouldn't be here, but I cannot disable `engine_cart` from inserting `webpacker` as a dependency.